contracts/token/non-fungible/src/non_fungible.rs

```diff
@@ -0,0 +1,343 @@
+    /// * `from` - Account of the sender.
+    /// * `to` - Account of the recipient.
+    /// * `token_id` - Token id as a number.
+    ///
+    /// # Errors
+    ///
+    /// * [`NonFungibleTokenError::IncorrectOwner`] - If the previous owner is
+    ///   not `from`.
+    /// * [`NonFungibleTokenError::UnauthorizedTransfer`] - If the caller does
+    ///   not have the right to approve.
+    /// * [`NonFungibleTokenError::NonexistentToken`] - If the token does not
+    ///   exist.
+    ///
+    /// # Events
+    ///
+    /// * topics - `["transfer", from: Address, to: Address]`
+    /// * data - `[token_id: u128]`
+    ///
+    /// # Notes
+    ///
+    /// We recommend using [`crate::transfer_from()`] when implementing this
+    /// function.
+    fn transfer_from(e: &Env, spender: Address, from: Address, to: Address, token_id: u128);
+
+    /// Gives permission to `to` to transfer `token_id` token to another
+    /// account. The approval is cleared when the token is transferred.
+    ///
+    /// Only a single account can be approved at a time.
+    /// To remove an approval, the owner can approve their own address,
+    /// effectively removing the previous approved address.
+    ///
+    /// # Arguments
+    ///
+    /// * `e` - Access to Soroban environment.
+    /// * `owner` - The address holding the tokens.
+    /// * `to` - Account of the recipient.
+    /// * `token_id` - Token id as a number.
+    /// * `live_until_ledger` - The ledger number at which the allowance
+    ///   expires.
+    ///
+    /// # Errors
+    ///
+    /// * [`NonFungibleTokenError::NonexistentToken`] - If the token does not
+    ///   exist.
+    /// * [`NonFungibleTokenError::InvalidApprover`] - If `auth` (param of
+    ///   [`Erc721::_approve`]) does not have a right to approve this token.
+    ///
+    /// # Events
+    ///
+    /// * topics - `["approval", from: Address, to: Address]`
+    /// * data - `[token_id: u128]`
+    ///
+    /// # Notes
+    ///
+    /// We recommend using [`crate::approve()`] when implementing this
+    /// function.
+    fn approve(e: &Env, owner: Address, to: Address, token_id: u128, live_until_ledger: u32);
+
+    /// Approve or remove `operator` as an operator for the owner.
+    ///
+    /// Operators can call [`Self::transfer_from`] for any token owned by the
+    /// caller.
+    ///
+    /// # Arguments
+    ///
+    /// * `e` - Access to Soroban environment.
+    /// * `owner` - The address holding the tokens.
+    /// * `operator` - Account to add to the set of authorized operators.
+    /// * `approved` - Flag that determines whether or not permission will be
+    ///   granted to `operator`. If true, this means `operator` will be allowed
+    ///   to manage `owner`'s assets.
+    /// * `live_until_ledger` - The ledger number at which the allowance
+    ///   expires.
+    ///
+    /// # Errors
+    ///
+    /// * [`NonFungibleTokenError::InvalidOperator`] - If `operator` is
```

Comment:
that's not possible

Reply:
addressed in [7552cf7](https://github.com/OpenZeppelin/stellar-contracts/pull/75/commits/7552cf74f0cf98b1838cc4af1e01e3451fd69904)